Source/iop/IopBios.cpp

```diff
@@ -50,8 +50,11 @@
 #define BIOS_MODULESTARTREQUEST_HEAD_BASE (CIopBios::CONTROL_BLOCK_START + 0x0018)
 #define BIOS_MODULESTARTREQUEST_FREE_BASE (CIopBios::CONTROL_BLOCK_START + 0x0020)
 #define BIOS_HANDLERS_BASE (CIopBios::CONTROL_BLOCK_START + 0x0100)
-#define BIOS_HANDLERS_END (BIOS_THREADS_BASE - 1)
-#define BIOS_THREADS_BASE (CIopBios::CONTROL_BLOCK_START + 0x0200)
+#define BIOS_HANDLERS_END (BIOS_PSEUDO_INTRHANDLER_TABLE_BASE - 1)
+#define BIOS_PSEUDO_INTRHANDLER_TABLE_BASE (0x480)

```

Comment:
Please call these `BIOS_SYSTEM_INTRHANDLER_TABLE_*`.
(ie.: `BIOS_SYSTEM_INTRHANDLER_TABLE_BASE` and `BIOS_SYSTEM_INTRHANDLER_TABLE_SIZE`)

Also, can you add a `static_assert` that makes sure that `BIOS_SYSTEM_INTRHANDLER_TABLE_BASE` is greater than `CIopBios::CONTROL_BLOCK_START`. This is just to make sure we don't forget to take this area in consideration if we ever change the start address of the control block.